packages/adapters/gupshup-whatsapp-v2/src/GupShupWhatsappAdapter.ts

```
@@ -0,0 +1,288 @@
+    xMsg.messageState = MessageState.REPLIED;
+    
+    const payload: ExtendedXMessagePayload = {
+      text: msg.text || ''
+    };
+    
+    xMsg.payload = payload;
+    
+    return xMsg;
+  }
+
+  async sendMessage(xMsg: XMessage): Promise<XMessage | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)